testsuite/MDAnalysisTests/utils/test_pickleio.py

```diff
@@ -0,0 +1,181 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
Just looking at codecov, there are RuntimeErrors for the __getstate__ of BZ2Picklable, FileIOPicklable, and GzipPicklable that aren't being covered here. Ideally it'd be great to cover them if they are reachable code paths.